playbooks/amc.yml

```diff
@@ -30,6 +30,7 @@
     - role: mongo
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
     - { role: 'rabbitmq', rabbitmq_ip: '127.0.0.1' }
+    # Do we need need edxapp 2 times here????
```

Comment:
Yes, you do need the two `edxapp` roles there. The first sets up the Celery works, the second sets up the main LMS/CMS processes.